bash_completion.sh

```diff
@@ -273,10 +273,10 @@ _buildtest ()
           esac
         ;;
         profiles)
-          local opts="--help -h list"
+          local opts="--help -h --remove -rm list"
```

Comment:
these will be commands so change `--remove` --> `remove` and `--rm` --> `rm`. Also make it alphabetically sorted